src/website/README.md

```
@@ -1 +1 @@
-# web-db
\ No newline at end of file
+# web-db .
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ' .'

(MD026, no-trailing-punctuation)